velox/type/TimestampConversion.cpp

```diff
@@ -327,8 +335,7 @@ bool tryParseDateString(
     }
   }
 
-  daysSinceEpoch = daysSinceEpochFromDate(year, month, day);
-  return true;
+  return daysSinceEpochFromDate(year, month, day, daysSinceEpoch).ok();
```

Comment:
Do we need to check if `daysSinceEpochFromDate` returns ok here?

Reply:
```
return daysSinceEpochFromDate(year, month, day, daysSinceEpoch).ok();
```
It will return true when status is ok, otherwise return false.